flowmachine/tests/test_cache_utils.py

```diff
@@ -0,0 +1,350 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Tests for cache management utilities.
+"""
+from cachey import Scorer
+from unittest.mock import Mock
+
+import pytest
+
+from flowmachine.core import Table
+from flowmachine.core.cache import (
+    compute_time,
+    shrink_below_size,
+    shrink_one,
+    size_of_cache,
+    size_of_table,
+    score,
+    get_query_by_id,
+    get_cached_queries_by_score,
+    touch_cache,
+    get_max_size_of_cache,
+    set_max_size_of_cache,
+    get_cache_half_life,
+    set_cache_half_life,
+    invalidate_cache_by_id,
+)
+from flowmachine.features import daily_location
+
+
+def test_scoring(flowmachine_connect):
+    """Test that score updating algorithm is correct by comparing to cachey as reference implementation"""
```

Comment:
Purely personal aesthetics-related comment, but I always find it more readable if the triple-quotes are on separate lines:
```suggestion
    """
    Test that score updating algorithm is correct by comparing to cachey as reference implementation
    """
```

I'll add this suggestion to the other test functions as well, but feel free to ignore if you disagree or if there is a style guide that recommends otherwise.

Reply:
Actually, I stopped halfway through because the Github interface is a little clumsy for this. 😂

Let me know if you're ok with this change and I'll reformat the remaining docstrings in a single commit.